### ISSUE TITLE
valid-jsdoc: Error when param has a default value

### ISSUE THREAD

**Issue by gyandeeps** [eslint/eslint#2528]
Nodejs: 0.12.2
Eslint: 0.21.0
Parser: default

```js
/**
 * This would be called to initiate all the file reads.
 * @param {string} [confFileName = package.json] - Configuration json file
 * @returns {void}
 * @public
 */
```

Error:

```sh
lib\fileManager.js
  1:0  error  Missing JSDoc parameter type for 'string'             valid-jsdoc
  1:0  error  Expected JSDoc for 'confFileName' but found 'string'  valid-jsdoc
```

I think this is a bug.

**Comment by nzakas** [comment]
I think you can't have spaces around the equals

**Comment by gyandeeps** [comment]
That is correct. Thanks.
